sct.py

```diff
@@ -425,7 +425,10 @@ def get_tags(ami):
         test_status = [click.style(k, fg='green') for k, v in test_status if v == 'PASSED'] + \
                       [click.style(k, fg='red') for k, v in test_status if not v == 'PASSED']
         test_status = ", ".join(test_status) if test_status else click.style('Unknown', fg='yellow')
-        tbl.add_row([ami.name, ami.image_id, ami.creation_date, tags['build-id'], test_status])
+        if not ami.name.startswith('releng-testing'):
+            tbl.add_row([ami.name, ami.image_id, ami.creation_date, tags['build-id'], test_status])
+        tbl.add_row([ami.name, ami.id, ami.creation_date, tags['build-id'], test_status])
```

Comment:
This line must be removed, otherwise you will print twice

Reply:
Removed